### ISSUE TITLE
Raspberry Pi support of partconv~ (ARM architecture)

### ISSUE THREAD

**Issue by corentoulf** [pd-l2ork/pd#21]
Hi,
I wanted to know if partconv~ could work on a Raspberry Pi ?
I tried to make it work but it looks like It's designed for Intel architecture... Is there any possibility to make it work on ARM ?

Thanks you !

**Comment by pd-l2ork** [comment]
I am not sure. Pd-L2Ork builds everything it can on RPi, so it is conceivable it is already built. I am not sure whether it actually works partially or at all. What library does the external belong to and did you find it missing on RPi build?

**Comment by corentoulf** [comment]
Thanks for your answer.
In fact, from my question, I did a fresh install of pd extended on a classic raspbian It is included in and it works, but with an issue at initialization : If I put an object called partconv~, it'll output an error about the architecture of the RPi, and nothing will work. If I put an object called bsaylor/partconv~ then it loads with no problems, and then I can create another partconv~ (no bsaylor/) working with no errors. 
I guessed it was an issue due to file path, but I tried different things about that, that didn't make it work at first load....
Strange behavior, but I could get it to work 😊
